### DISCUSSION TITLE
in_forward input errors

### DISCUSSION THREAD

**Discussion by nagarnar** [fluent/fluentd#4083] [General]
I have fluentbit sending prometheus_scrape metrics using forward output plugin to be received by the aggregator fluentd. And on some data coming I get this error on fluentd

`2023-03-07T23:22:35.693287950Z 2023-03-07 23:22:35 +0000 [warn]: #0 fluent/log.rb:351:warn: emit transaction failed: error_class=TypeError error="can't convert Hash into an exact number" location="<internal:timev>:272:in `at'" tag="fluentbit-metrics"`

**Comment by daipom** [comment]
I can't be certain with that information, but it appears that the data transferred to `in_forward` was not in an expected format.

Please confirm if fluentbit sends the data in the correct format for Fluentd's `in_forward`.
